ci/Dockerfile

```diff
@@ -1,17 +1,11 @@
-FROM ubuntu:18.04
+FROM ubuntu:21.04
```

Comment:
wow ... so client-c's image is not centos7?

Reply:
This image is for client-c CI testing only.

There is no such "client-c image".